jenkins.yml

```diff
@@ -22,6 +22,22 @@
     sudo_defaults:
       - defaults: '!requiretty'
 
+  pre_tasks:
+    - name: Add jenkins service unit to systemd in case of docker.
+      copy:
+        dest: /etc/systemd/system/jenkins.service
+        content: |
```

Comment:
Additionally, would it make more sense to actually add this directly to the Jenkins role instead and get this pushed upstream?